packages/amplify-cli/package.json

```diff
@@ -9,252 +9,38 @@
   },
   "author": "Amazon Web Services",
   "license": "Apache-2.0",
-  "main": "lib/index.js",
+  "main": "binary.js",
   "types": "lib/index.d.ts",
   "keywords": [
     "graphql",
     "appsync",
     "aws"
   ],
   "bin": {
-    "amplify": "bin/amplify"
+    "amplify": "./run.js"
   },
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
-    "test": "jest",
-    "postinstall": "node scripts/post-install.js",
-    "build": "tsc",
-    "watch": "tsc -w",
-    "clean": "rimraf ./lib tsconfig.tsbuildinfo"
+    "postinstall": "node ./install.js || echo \"failed to install amplify binary\"",
+    "run": "node ./run.js",
+    "fmt": "prettier --write **/*a.js",
+    "fmt:check": "prettier --check **/*.js"
   },
+  "binaryLocation": "https://d380lpju5pcsmv.cloudfront.net",
```

Comment:
this is the cloudfront distribution url where the binaries are currently being uploaded.